addons/pos_self_order/controllers/manifest_controller.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import base64
+import json
+
+from odoo import http
+from odoo.http import request
+from odoo.tools import ustr, file_open
+
+
+class PosManifest(http.Controller):
+
+    @http.route('/pos-self/manifest', type='http', auth='public', methods=['GET'])
+    def posmanifest(self):
+        """ Returns a WebManifest describing the metadata associated with a web application.
+        Using this metadata, user agents can provide developers with means to create user
+        experiences that are more comparable to that of a native application.
+        """
+        company_name = request.env.company.name
+        company_id = request.env.company.id
+        manifest = {
+            'name': 'Self Order',
+            'short_name': '%s' % company_name,
+            'scope': '/pos-self',
+            'start_url': '/pos-self',
+            'display': 'standalone',
+            'background_color': '#714B67',
+            'theme_color': '#714B67',
+            'orientation': 'portrait-primary'
+        }
+        icon_sizes = ['256x256']
```

Comment:
It's good to have provided for the possibility of different icon sizes. Because there are different specificities depending on the OS?

Can you see about adding them? I'm thinking in particular of Apple, which must be pretty demanding.

PWA General: [link](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/How_to/Define_app_icons)
Microsoft: [link](https://learn.microsoft.com/fr-fr/microsoft-edge/progressive-web-apps-chromium/how-to/icon-theme-color)
Android: [link](https://developer.android.com/distribute/google-play/resources/icon-design-specifications?hl=fr)
Apple: [link](https://developer.apple.com/design/human-interface-guidelines/app-icons#App-icon-sizes)